servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/HttpMessageDiscardWatchdogServiceFilterTest.java

```diff
@@ -62,6 +66,11 @@ final class HttpMessageDiscardWatchdogServiceFilterTest {
     @BeforeEach
     public void setup() {
         loggerStringWriter.reset();
+        // Ensure our logger is fully initialized.
+        String expected = "Logger initialized";
+        do {
+            LOGGER.info(expected);
+        } while (!loggerStringWriter.accumulated().contains(expected));
```

Comment:
We could probably generalize this to the `LoggingStringWriter` itself if it proves to be the problem. I was hesitant to do so since I hadn't noticed other tests that depend on it being flaky as well.

Reply:
Consider adding at least a small sleep in the loop (1-10ms) which will probably not hurt the runtime but give other code a chance to run more quickly (i.e. the logger init code?)